quick_sort.py

```diff
@@ -0,0 +1,40 @@
+import timeit
+import random
+
+
+def q_sort(self):
+    lower = []
+    p_list = []
+    higher = []
+    if len(self) <= 1:
```

Comment:
considering that the first three symbols you declare are useless if this test passes, you might put the escape clause first, and then start setting up for a real sort.